### PR TITLE
(docs: FAQ) Added code example to demonstrate moving of windows acros…

### PR DESCRIPTION
**Pull request by Arjun-Somvanshi**
I had trouble finding the solution to being able to move windows across various groups on different screens, the code here can help with that.

### CODE REVIEW

File: docs/manual/faq.rst [hunk 1]
```diff
@@ -102,6 +102,27 @@ of binding keys to ``lazy.group[name].toscreen()``, use this:
...
+
+.. code-block:: python
+    
+    def go_to_group_and_move_window(name: str):
+        def _inner(qtile):
+            if len(qtile.screens) == 1:
+                qtile.current_window.cmd_togroup(name, switch_group=True)
+                return
+
+            if name in '1234':
```
Comment (m-col) [inline docs/manual/faq.rst:116]: Just to match the groups used in the previous example
```suggestion
            if name in "123":
```

File: docs/manual/faq.rst [hunk 2]
```diff
@@ -102,6 +102,27 @@ of binding keys to ``lazy.group[name].toscreen()``, use this:
 
     for i in groups:
         keys.append(Key([mod], i.name, lazy.function(go_to_group(i.name))))
+        
+To be able to move windows across these groups a similar function can be used:
```
Comment (m-col) [inline docs/manual/faq.rst:106]:
```suggestion
To be able to move windows across these groups which switching groups, a similar function can be used:
```

**Review by m-col — CHANGES_REQUESTED** [review]
Hey sorry looks like we all missed this one. Looks good, thanks for this! I just made a couple tiny suggestions :)

**Comment by Arjun-Somvanshi** [comment]
It says the screenshot widget has failed to build. Let me know what I need to do to resolve this. Thanks.

**Comment by m-col** [comment]
The CI failure is unrelated. This change won't cause any failures. Thanks again!

**Pull request merged**